airflow/cli/commands/standalone_command.py

```diff
@@ -235,13 +236,13 @@ def port_open(self, port):
             return False
         return True
 
-    def job_running(self, job):
+    def job_running(self, job_runner_class: type[BaseJobRunner]):
         """
         Checks if the given job name is running and heartbeating correctly.
 
         Used to tell if scheduler is alive.
         """
-        recent = job.most_recent_job()
+        recent = job_runner_class.most_recent_job()
```

Comment:
Why. I am happy to make changes even in trivial things. 

Yes. I had this initially exactly this way, but I found that this has a bit of a problem for those who got used to XXXJob.most_recent_job (polimorhism made sure that we were selecting the "right" job_type. It feels a bit better to run this query with "class" context rather than pass the context to is. But I will see - maybe coupled with making the Job type class variable it would make sense actually.

Reply:
Moved it back to base_job (but as a function not a class method). I will still have to fix the lazy loading on health_check for that one ( some tests are failing and I need to look at them) - but it should be better now.

Reply:
OK. I changed the approach here:

* moved it back to BaseJob (but also moved it outside of the class)
* I am passing now BaseJobRunner type as parameter to determine the job type
* the status of scheduler/triggerer is now displayed via lazy-proxied lambda performing the query with the right parameter - rather than lazy-proxied method of the Job-derived object.

Looks like it works nicely and is more logical.